analyzer/src/funTest/kotlin/SbtTest.kt

```diff
@@ -31,6 +31,9 @@ import io.kotlintest.specs.StringSpec
 
 import java.io.File
 
+import kotlinx.coroutines.ObsoleteCoroutinesApi
```

Comment:
Left-over?

Reply:
I'm using a different way to create the dispatcher now, not using the obsolete API anymore.

Reply:
Sounds great! Are you planning to also update the existing parallel scanner code accordingly (in a new PR, of course)?

Reply:
Ah, I just saw your `scanner-no-obsolete-api` branch, perfect :-)